src/test/scala/castalia/StubConfigParserSpec.scala

```diff
@@ -0,0 +1,63 @@
+package castalia
+
+import java.io.FileNotFoundException
+
+import castalia.model.{CastaliaConfig, ResponseConfig}
+
+//import castalia.model.{StubConfig}
+import org.scalatest.{Matchers, WordSpec}
+import spray.json.{JsString, JsValue}
+
+/**
+  * Created by mihaelaoprea on 02/12/15.
+  */
+class StubConfigParserSpec extends WordSpec with Matchers {
+
+  "A StubConfigParser" when {
+    "json file \"jsonconfiguredstub.json\" exists on classpath" should {
+      "return a StubConfig object" in {
+
+        val stub = StubConfigParser.parseStubConfig("jsonconfiguredstub.json")
+
+        stub.endpoint shouldBe "jsonconfiguredstub"
+        val responses: List[ResponseConfig] = stub.responses
+        responses.length === 3
+
+        val req1Response = responses.find(r => r.id == "1").get

```

Comment:
If this is an `Option`, we can also use `.loneElement` (for which we have to mix in `OptionValues` (http://www.scalatest.org/user_guide/other_goodies))